release-notes.md

```diff
@@ -16,12 +16,14 @@
 	moduls.enableStaticContentConversion.basePermissionType =
 	moduls.enableStaticContentConversion.basePermissionName =
 	moduls.enableStaticContentConversion.group = GROUP_ENABLE_DISABLE_FEATURES
+  
 
 ### Deployment Scripts ### 
 	php deployment/updates/scripts/2021_07_26_deploy_capture_tools_content_conversion_data.php
 	php deployment/updates/scripts/2021_07_26_deploy_kaltura_meeting_recordings_conversion_data.php
 	php deployment/updates/scripts/2021_07_26_deploy_msft_teams_recordings_conversion_data.php
 	php deployment/updates/scripts/2021_07_26_deploy_zoom_recordings_conversion_data.php
+    php /opt/kaltura/app/deployment/updates/scripts/add_permissions/2021_08_03_service_baseentry.php
```

Comment:
You should add new section for this item.
Adding Jira, describing the feature + deployment script.